diff --git a/packages/compass-shell/src/stores/store.ts b/packages/compass-shell/src/stores/store.ts
index 9eafd29d1..7217a2162 100644
--- a/packages/compass-shell/src/stores/store.ts
+++ b/packages/compass-shell/src/stores/store.ts
@@ -3,7 +3,8 @@ import type { AnyAction } from 'redux';
 import type { ThunkAction } from 'redux-thunk';
 import type { WorkerRuntime } from '@mongosh/node-runtime-worker-thread';
 import { ObjectId } from 'bson';
-import { createWorkerRuntime } from '../modules/worker-runtime';
+// @ts-ignore
+import { createWorkerRuntime } from 'worker-runtime';
 import type { ShellPluginExtraArgs } from '../plugin';
 
 /**
